### ISSUE TITLE
[Feature] Lock fields

### ISSUE THREAD

**Issue by echo6ix** [stashapp/stash-box#213]
Eventually Stashbox is going going to encounter cases of vandalism or a lot of good faith bad edits on the same recurring fields of data. While all edits require approval, the pending edits will still clutter the queue. I think this is where locked fields can be useful. 

[TMDB](https://www.themoviedb.org/bible/general/59f792a29251413e93000001#59f793929251413e93000010) defines locked fields as,

>Confirmed data is routinely locked by content moderators. Moderators may also lock data to prevent vandalism.
>When you stumble upon incorrectly locked data, please use the report button, explain the problem and link us to a proper source if needed.

Locked fields are indicated by a padlock icon next to the field. Unlocked fields are indicated by an open padlock next to the field.

**Comment by xiosensei** [comment]
instead of a padlock icon. I vote for a check mark for verified data. and no check mark for unverified data.

**Comment by echo6ix** [comment]
Utilizing a padlock icon and the term _locked_ offers greater adaptability as a symbol of a locked state. For example using "verified" becomes incongruous when applied to securing fields in order to prevent acts of vandalism. Yes, fields should be locked when we have confidence that they're immutable or accurate through our own internal processes, but "verified" conveys a higher level of certainty and rigor about the data.

TLDR both "locked" and "verified" convey some level of validation, but "verified" conveys a stronger guarantee of accuracy and it is also not applicable when referring to a locked state for other reasons such as vandalism.

**Comment by echo6ix** [comment]
Locked field report button
1. There are going to be fields that get locked erroneously because mistakes happen
   - Lets give users a chance to report these mistakes in a streamlined efficient fashion w/o having to go through Discord.
   - If a status indictor such as a padlock is to be used to indicate a locked field, perhaps have this padlock icon can be a link that directs the user to the relevant comment form; prints out an issue-esque template similar to Github for the user to make their case, and notifies the mods. Or it could just open a modal window with the adequate fields for the user to populate which then gets pushed to the comment section and notifies the mods
2. Conversely there might be fields that should be locked but are not
   - Perhaps offer a disabled/greyed-out padlock icon beside most fields that the users can interact with similar to the above, where they can suggest fields that should be locked
   - This feature should probably be experimental. I suspect it might get annoying and could be open for abuse, but who knows

Of course these features are contingent on some kind of notification system, which I am not sure StashDB has. But I am documenting them for prospective purposes.

**Comment by AdultSun** [comment]
An alternative to the "report" button for locked fields is to allow edits to be made to locked fields (manual only, not via draft) and send them through the edit queue, but require a mod/admin to approve those changes before they can pass. It would still open up those edits to scrutiny and feedback from the larger pool of editors but still give mods/admins the final decision.
